tests/virt_autotest/guest_installation_run.pm

```diff
@@ -64,6 +65,18 @@ sub post_execute_script_assertion {
 sub run {
     my $self = shift;
 
+    # refer to poo#135431 for more details
+    my $guest = get_var('GUEST_PATTERN');
+    if ($guest =~ m/sles-15-sp6/i) {
+        my $proxyscc = get_var('SCC_URL');
+        my $fv_autoyast = "/usr/share/qa/virtautolib/data/autoinstallation/sles/15/sp6/64/fv/def";
+        script_run("sed -i '8i\ <reg_server>$proxyscc</reg_server>' $fv_autoyast");
+        if (is_xen_host) {
+            my $pv_autoyast = "/usr/share/qa/virtautolib/data/autoinstallation/sles/15/sp6/64/pv/def";
+            script_run("sed -i '8i\ <reg_server>$proxyscc</reg_server>' $pv_autoyast");
+        }
+    }
```

Comment:
Actually this can be simplified also as below:
```
my $autoyast = "/usr/share/qa/virtautolib/data/autoinstallation/sles/15/sp6/64/" + (is_xen_host ? pv : fv ) + "/def";
my $proxyscc = get_var('SCC_URL');
script_run("sed -i '8i\ <reg_server>$proxyscc</reg_server>' $autoyast");
```

Reply:
Cool! Had made code updated